packages/babel-cli/src/babel/dir.js

```diff
@@ -7,21 +7,31 @@ import fs from "fs";
 
 import * as util from "./util";
 
+const NOT_COMPILABLE = null;
```

Comment:
I'd prefer `NOT_COMPILABLE = {}`, so that its value is unique.

Reply:
Yeah, I wasn't sure what the best way to do this is. That works.